frontend/src/styles/loader.sass

```diff
@@ -20,6 +20,8 @@ default-loader
 .loader
   $side: 1em
   color: #050505
+  position: fixed !important
```

Comment:
Why you need important here...

Reply:
to override the value from the library, otherwise it is not overridden